docs/en/ingest-management/elastic-agent/run-container-common/download-elastic-agent.asciidoc

```diff
@@ -31,6 +31,9 @@ inputs:
 ------------------------------------------------
 endif::[]
 
+For KSPM purposes, the {agent} requires read access to various types of Kubernetes resources, node processes, and files.
+To achieve this, read permissions are granted to the {agent} for the necessary resources, and volumes from the hosting node's file system are mounted to allow accessibility to the {agent} pods
```

Comment:
I'm assuming that passive voice is used here because this happens without the user having to do anything extra beyond the steps documented in this topic? If that's true, this sentence is fine as-is.

Reply:
Yes, you are right, a user that wants to install elastic-agent on Kubernetes downloads an installation file that will install the elastic-agent with the permissions I described.